instrumentation/opentelemetry-instrumentation-requests/tests/test_requests_integration.py

```diff
@@ -472,3 +474,46 @@ def perform_request(url: str, session: requests.Session = None):
         request = requests.Request("GET", url)
         prepared_request = session.prepare_request(request)
         return session.send(prepared_request)
+
+
+class TestRequestsIntergrationMetric(TestBase):
+    URL = "http://httpbin.org/status/200"
+
+    def setUp(self):
+        super().setUp()
+        RequestsInstrumentor().instrument(meter_provider=self.meter_provider)
+
+        httpretty.enable()
+        httpretty.register_uri(httpretty.GET, self.URL, body="Hello!")
+
+    def tearDown(self):
+        super().tearDown()
+        RequestsInstrumentor().uninstrument()
+        httpretty.disable()
+
+    @staticmethod
+    def perform_request(url: str) -> requests.Response:
+        return requests.get(url)
+
+    def test_basic_metric_success(self):
+        self.perform_request(self.URL)
+
+        expected_attributes = {
+            "http.status_code": 200,
+            "http.url": self.URL,
+            "http.host": "httpbin.org",
+            "http.method": "GET",
+            "http.flavor": "1.1",
+            "http.scheme": "http",
```

Comment:
Are we missing a few? `net.*`, `http.target`?

Reply:
Added `net.*` attributes. 
I have not added `http.target` as the target is not parameterised and adding raw target can lead to high cardinality.
Its similar for `http.url` as discussed [here](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1110#discussion_r890852991)